content/docs/1.2.3/deploy/install/_index.md

```diff
@@ -86,8 +86,41 @@ If MountPropagation is disabled, Base Image feature will be disabled.
 
 ### Root and Privileged Permission
 
-The longhorn manager uses `Bidirectional` mount propagation to access the host's disk (default `/var/lib/longhorn`) in the container.
-An use case is to access disk metadata and replicas directory on the host.
+The longhorn components need to access the host's disk in the container.
+A use case is to access disk metadata and replicas directory on the host. The list of the directories of all components are:
```

Comment:
Below are examples to list the different use cases requiring root access and privileged permissions.